src/log-manager/src/conf/nginx.conf

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) Microsoft Corporation
+# All rights reserved.
+#
+# MIT License
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+# documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+# to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+# The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+# BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+# NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+# DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+worker_processes  1;
+
+events {
+  worker_connections  1024;
+}
+
+http {
+  include            mime.types;
+  default_type       application/octet-stream;
+  sendfile           on;
+  keepalive_timeout  65;
+
+  resolver 127.0.0.1;
+
+  server {
+    listen      80;
+    server_name localhost;
+    client_max_body_size 0; # Disable checking of client request body size.
+    client_body_buffer_size 256M;
+    proxy_connect_timeout 60m;
+    proxy_send_timeout 60m;
+    proxy_read_timeout 60m;
+    send_timeout 60m;
+
+    #
+    # Health check.
+    #
+    location = /healthz {
+      default_type text/plain;
+      return 200 "Log manager ready.";
+    }
+
+    #
+    # Get log for job
+    #
+    location /logs {
+      alias /usr/local/pai/logs;
```

Comment:
add `default_type text/plain`?

it seems alias does not work

Reply:
It works, just not deployed on cluster